UnityProject/Assets/Scripts/Messages/Server/GhostRoles/GhostRoleUpdateMessage.cs

```diff
@@ -39,6 +40,7 @@ public static NetMessage SendToDead(uint key)
 
 				foreach (ConnectedPlayer player in PlayerList.Instance.InGamePlayers)
 				{
+					if(PlayerList.Instance.loggedOff.Contains(player)) continue;
```

Comment:
This is unnecessary `PlayerList.Instance.InGamePlayers` is only loggedIn players.

`public List<ConnectedPlayer> InGamePlayers => loggedIn.FindAll(player => player.Script != null);`

Reply:
I'll add a comment to that variable so we can know that in the future